srtcore/common.cpp

```diff
@@ -991,6 +991,6 @@ std::string LogDispatcher::Proxy::ExtractName(std::string pretty_function)
     return pretty_function.substr(pos+2);
 }
 
-} // (end namespace srt_logging)
-
 #endif
+
+} // (end namespace srt_logging)
```

Comment:
I am thinking if it is better to move `#if ENABLE_LOGGING` to line 841 instead (right above the namespace declaration).

Reply:
Right. I think it's better to move the `#if ENABLE_LOGGING` up to the place before `namespace srt_logging` starts. None of these are required when logging is not enabled.